app/src/main/java/com/github/factotum_sdp/factotum/ui/roadbook/DRecordList.kt

```diff
@@ -118,6 +189,48 @@ class DRecordList(
      * @return a new DRecordList with the main (this) list displaying only the non-archived records
      */
     fun withoutArchived(): DRecordList {
-        return DRecordList(allRecords, archived, false)
+        return DRecordList(allRecords, archived, timestamped, false)
+    }
+
+
+    /** Helpers : */
+    private fun computeTimestampedOnAdd(recordToAdd: DestinationRecord): Set<DestinationRecord> {
```

Comment:
Possible way to refactor this, like you call it computeTimestamped(action_add : boolean) or something like